app/widgets/common/deployModal/EnvironmentDropdown/EnvironmentDropdownList.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useMemo } from 'react';
+import { isEmpty } from 'lodash';
+import { Form } from 'cloudify-ui-components';
```

Comment:
As we are actually not using here any `Form` related components other than `Dropdown`, we could probably just:
```suggestion
import { Dropdown } from 'cloudify-ui-components';
```
WDYT?

Reply:
Sure 👌 
To be honest, till this point I didn't know that form components could be imported explicitly so it's good to learn new stuff 😁

Reply:
Fixed as a part of 79424d22fda153f392b5ca7487655319edf901de 🎉